src/main/java/com/google/sps/servlets/MemesServlet.java

```diff
@@ -87,5 +92,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String imgTag = String.format("<img src=\"%s\" width = \"300\"/>", blob.getMediaLink());
       response.getWriter().println(imgTag);
     }
+  }
+
+/** This funtions  randomly  get a string  captions from an array of string. When the function is called
+ * it randomly return memes caption from an array
+ * input: null
+ * return: strings..
+ * 
+ */
+  
+
+  public static String getCpation(){
```

Comment:
I think that you should go through your code and fix spelling errors in method names or tags that you will call later on. [Like getCpation( ) --> getCaption( ) ] If it remains misspelled, other may have a hard time remembering or getting the method signature right when working with them. There are some words misspelled in the commenting but I think that's ok, just consider fixing the method names, variable names and such.

Reply:
+1 Good explanation, Ximena!